libs/blocks/tabs/tabs.js

```diff
@@ -84,6 +86,68 @@ function initTabs(elm, config, rootElem) {
   if (config) configTabs(config, rootElem);
 }
 
+function previousTab(item, i, arr) {
+  const next = arr[i + 1];
+  return (!isTabInTabListView(item) && next && isTabInTabListView(next, true));
+}
+
+function nextTab(item, i, arr) {
+  const previous = arr[i - 1];
+  return (!isTabInTabListView(item) && previous && isTabInTabListView(previous, true));
+}
```

Comment:
odd to have `nextTab()` contain and check for the const `previous` and vice versa with `previousTab()` and `next`

Reply:
I thought about documenting this as the logic is a little hard to parse from the expression.
previousTab is looking for the first tab not in view with the next tab in view, example tab1 not in view with tab2 in view.
nextTab is looking for the last tab not in view with the previous tab in view, example tab2 not in view with tab1 in in view.